app/src/main/java/com/github/giommok/softwaredevproject/Database.java

```diff
@@ -64,11 +64,8 @@ public void onDataChange(DataSnapshot snapshot) {
 
             @Override
             public void onCancelled(@NonNull DatabaseError error) {
-                throw new DatabaseException();
             }

```

Comment:
Handle what needs to be done in the onCancelled callback before merging to master

Reply:
Actually, there is nothing to handle: onCancelled is triggered when the permission to read/write data is denied and on the official tutorial they suggested putting a simple log instruction. As we don't need that, I would prefer to have nothing (but I can't remove it as the listener requires the onCancelled method).